sound/soc/sof/Makefile

```diff
@@ -6,7 +6,7 @@ ifneq ($(CONFIG_SND_SOC_SOF_CLIENT),)
 snd-sof-objs += sof-client.o
 endif
 
-snd-sof-$(CONFIG_SND_SOC_SOF_DEBUG_PROBES) += probe.o compress.o
+snd-sof-$(CONFIG_SND_SOC_SOF_DEBUG_PROBES) += sof-probe.o
```

Comment:
I am good with this as long as this doesn't interfere with @dbaluta's work to support real compressed streams (MP3, etc).

Reply:
Thanks, this is the reason I have created this draft, to make sure that this is aligned with what @dbaluta is planning.
Also it was a bit confusing for me as a newcomer. I thought that we already have compressed support, which turned out to be not true.

Reply:
@ujfalusi thanks for creating the draft. I think it the best way to proceed is for you to rebase your work on https://github.com/thesofproject/linux/pull/2703

Now that my work on firmware side is mostly done I will go back and adress the issues with #2703. Should be ready by the end of the week.

Reply:
@dbaluta, with a quick look at #2703 we are doing essentially the same thing, I go a bit further by merging the two files, but I can just rebase on your work.